doc/classes/RenderingDevice.xml

```diff
@@ -1583,22 +1583,22 @@
 		<constant name="BARRIER_MASK_VERTEX" value="1" enum="BarrierMask" is_bitfield="true">
 			Vertex shader barrier mask.
 		</constant>
-		<constant name="BARRIER_MASK_FRAGMENT" value="2" enum="BarrierMask" is_bitfield="true">
+		<constant name="BARRIER_MASK_FRAGMENT" value="8" enum="BarrierMask" is_bitfield="true">
 			Fragment shader barrier mask.
 		</constant>
-		<constant name="BARRIER_MASK_COMPUTE" value="4" enum="BarrierMask" is_bitfield="true">
+		<constant name="BARRIER_MASK_COMPUTE" value="2" enum="BarrierMask" is_bitfield="true">
 			Compute barrier mask.
 		</constant>
-		<constant name="BARRIER_MASK_TRANSFER" value="8" enum="BarrierMask" is_bitfield="true">
+		<constant name="BARRIER_MASK_TRANSFER" value="4" enum="BarrierMask" is_bitfield="true">
 			Transfer barrier mask.
 		</constant>
-		<constant name="BARRIER_MASK_RASTER" value="3" enum="BarrierMask" is_bitfield="true">
+		<constant name="BARRIER_MASK_RASTER" value="9" enum="BarrierMask" is_bitfield="true">
 			Raster barrier mask (vertex and fragment). Equivalent to [code]BARRIER_MASK_VERTEX | BARRIER_MASK_FRAGMENT[/code].
 		</constant>
-		<constant name="BARRIER_MASK_ALL_BARRIERS" value="15" enum="BarrierMask" is_bitfield="true">
+		<constant name="BARRIER_MASK_ALL_BARRIERS" value="32767" enum="BarrierMask" is_bitfield="true">
```

Comment:
Yes, all barriers means all barriers, so I think we need to update the documentation to reflect this.

That said, seeing future enhancements will likely purely mean that raster will be broken up further, or maybe that there will be some future reason that compute needs to be divided up it will probably still cover the description.

All in all, the likelihood of further enums being introduced is very tiny, this is purely a matter of "if the value changes anyway, might as well change it to something so we don't get into this situation again, even if that will never happen".

Reply:
Let's update the docs in this PR before merging

Reply:
Updated!